tools/dive.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Dive",
+    "description": "A tool for exploring a docker image, layer contents, and reverse engineering the docker image - layer by layer.",
+    "tags": [
+        "containers",
+        "devops",
+        "automation",
+        "Security",

```

Comment:
Remove space with `-` and also lowercase letters?

In future, will handle this at pipeline with linting and formatting.